wazuh/wazuh-dashboard-plugins
# Add macOS log collector tab

### Description
This PR adds macOS log collector tab.
 
### Issues Resolved
Closes #6389 

### Evidence
![image](https://github.com/wazuh/wazuh-dashboard-plugins/assets/9343732/fb40a619-3773-461b-91d1-42eb251cabf1)



### Test
- Install a macOS agent
- Configured log collector - [Documentation](https://documentation.wazuh.com/current/user-manual/capabilities/log-data-collection/configuration.html#collecting-macos-uls-logs-with-the-wazuh-agent)
- Go to Endpoints Summary
- Select the macOS agent
- Go to Configuration / Log collection
- Check the macOS tabs exists and the configuration is correct
![image](https://github.com/wazuh/wazuh-dashboard-plugins/assets/9343732/7b7a7a87-3d57-4998-936e-802e8aff57de)


### Check List
